buzzfeedbotOS.py

```diff
@@ -19,45 +20,25 @@ def reddit_bot(headline, main_text, link):
 '''Checks if there are numbered bullet points in the article. If there is at least one, returns true, 
 if there are none, returns false''' 
 def check_for_numbered_points(link_to_check):
-	i=0
 	session = requests.Session()
 	clickbait_article = session.get(link_to_check)
 	soup = BeautifulSoup(clickbait_article.content, 'html.parser')
-	for title in soup.find_all('h3'):
-		for number in title.find_all('span', attrs={'class': 'subbuzz__number'}):
-			i+=1
-	if i > 0:
-		return (True)
-	else:
-		return (False)
+	return any(title.find_all('span', attrs={'class': 'subbuzz__number'}) for title in soup.find_all('h3'))
 		
 '''Gets current time in EST'''
 def current_time_eastern():
-	from time import gmtime, strftime
-	time_diff = 4
-	gm_time = strftime("%#H:%M:%S", gmtime())
-	gm_hour, min_secs =  gm_time.split(':', 1)
-	if int(gm_hour)>= time_diff:
-		est_hour = str(int(gm_hour)-time_diff)
-	else:
-		est_hour = str(int(gm_hour)+24-time_diff)
-	es_time = est_hour + ':'+ min_secs
-	return es_time
+	now_est = datetime.now(pytz.timezone('EST'))
+	return '{}:{}:{}'.format(now_est.hour, now_est.minute, now_est.second)

```

Comment:
Time is **hard** when programming. For example, what happens on daylight saving time, or leap seconds? Prefer to use established libraries for this.